src/pages/Layout.jsx

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import { Outlet } from 'react-router-dom'
-import Navbar from './Navbar'
+// import Navbar from './Navbar'
 
 const Layout = () => {
   return (
     <>
-      <Navbar />
+      {/* <Navbar /> */}
```

Comment:
Comment: Same as above, if not going to use shortly, then best to delete for now. If you really want to keep it commented out then I'd suggest to leave a comment in the code as to why it's commented out and not deleted.